openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -271,6 +271,21 @@ def process(self, instance):
             ["DEADLINE_REST_URL"]
         )
 
+        self._job_info = (
+            context.data["project_settings"]
+            ["maya"]
+            ["publish"]
+            ["deadline"]
+            ["jobInfo"]
```

Comment:
I would add `.get("jobInfo", {})` for cases that settings are not updated yet (Same for `"pluginInfo"`).